Master

phil-fradkin opened the pull request.

thejmazz committed:
filterByBad/Good functions, iterative COdat filtering examples, COdata.R, some plots

thejmazz committed:
Merge branch 'master' of github.com:hyginn/Ontoscope

thejmazz committed:
Update README.md

thejmazz committed:
Delete phylify.R

biodim committed:
fantomSummarize() fix

biodim committed:
Added Profiler Output

Commit:
Changed columns and filename

Changed columns to CellFrom and CellTo
Made Rdata filename match R variable name

Commit:
Merge branch 'master' of https://github.com/hyginn/Ontoscope

Commit:
Added files via upload

thejmazz committed:
super mini p.o.c gather

thejmazz committed:
Merge branch 'master' of github.com:hyginn/Ontoscope

kartikaychadha committed:
Rename contast.R to contrast.R

ontoscoper committed:
added more generic split_col_across_rows()

splits the specified column of fantomResults across multiple new otherwise
column-duplicated rows

thejmazz committed:
Merge branch 'Adding-asses'

thejmazz committed:
Merge branch 'master' of github.com:hyginn/Ontoscope

ontoscoper committed:
revert stated location of Sample_DB.txt so fantom_main.R can find it

hyginn committed:
Begin code to integrate modules

hyginn committed:
Added comment

hyginn committed:
Raw data for TRRUST GRN

hyginn committed:
remove

hyginn committed:
remove

biodim committed:
fixed working directory bug for TRRUST

biodim committed:
fantom_import V 1.0.0

biodim committed:
Update README.md

biodim committed:
updated runOntoscope for TRRUST

thejmazz commented:
can you just commit directly to master, none of these PRs have any commits from you too

biodim committed:
added Reverse Search

JialunC committed:
enrichr v1.0

hyginn committed:
Code comments

hyginn committed:
CR

hyginn committed:
Code comments

thejmazz committed:
initial nicePath

kartikaychadha committed:
contrast version 0.3 release

ontoscoper committed:
Revert "contrast version 0.3 release"

This reverts commit 0c16328ff9f09f215fe7f9da42f7a92c2c424da8.

kartikaychadha committed:
Version 0.3 Contrast 

Details on Readme file

kartikaychadha committed:
Contrast Version 0.3 Details added

ontoscoper committed:
sped up an operation in fantomSummarize() from ~10.5 seconds on my machine to
0.01 s

also eliminated the side effect that fantomSummarize() was previously modifying
fantomResults by calling deleteEmpty()

ontoscoper committed:
Merge branch 'experimental'

hyginn committed:
setting n=1 as parmeter default; some style adjustments; code audit comments

hyginn committed:
style updates and code audit comments

JialunC committed:
enrichr v2.0 with a quantitative expression dataset as  input

JialunC committed:
Merge branch 'master' of https://github.com/hyginn/Ontoscope

SKamdar1 committed:
Literature Confirmation - Analyze Submodule

Background files (pre-generated for your convenience!) and script to
obtain confidence ranks and scores for output transcription factor
lists. Currently using dummy data for output TF file and origin/target
search info.

anamqu committed:
Added files via upload

eugeniabarkova committed:
GATHER Code

For some reason doesn't want to get committed since the morning :(

SKamdar1 committed:
Merge branch 'master' of https://github.com/hyginn/Ontoscope

gosuzombie committed:
added first iteration of gene set enrichment analysis, missing method headers and annotation

gosuzombie committed:
Delete qselslvq.lso.txt

biodim committed:
Restored Deleted/Modified Files

kartikaychadha committed:
V 0.3.2: Updating contrast.R

Specifications added along with comments.
Verification file code in comments. See codes

thejmazz committed:
new COdat not filtering out human 'time series' or 'fractionations and perturbations'

thejmazz committed:
Merge branch 'master' of github.com:hyginn/Ontoscope

biodim committed:
All 1829 Samples are Now Matchable

burtonlm committed:
Latest  version of RANK

burtonlm committed:
Added files via upload

thejmazz committed:
Merge branch 'master' of github.com:hyginn/Ontoscope

Commit:
updated assess

SKamdar1 committed:
V 1.1 of PubmedMiner.R

Added tests and checks for TFs which may not have any Pubmed abstract hits or invalid TFs.

anamqu committed:
Added files via upload

Commit:
refine STRING network

Further refine the STRING subnetwork to include all and only genes for
which we also have expression data.

Commit:
Added genes returned from Normalize_TF module

biodim committed:
Update README.md

biodim committed:
Update README.md

biodim committed:
V 1.0.2

eugeniabarkova committed:
Updated GATHER to include a method for multiple targets

Also, fixed a couple of bugs and added proper readme.

phil-fradkin committed:
Final version of asses 

Manually change Interaction graph and add gsx scores (loaded from /annotate trial dataset)
Parent edge calculation (multiple not just transcription factor)

sonarestis committed:
WEAVE 0.3

Incorporated REGNET and TRRUST networks into WEAVE. Downstream modules can now just call this module to obtain igraph objects for the respective database

sonarestis committed:
Added comments to code for better understanding

phil-fradkin committed:
Updates assess

Added complexity to calculation of distance scores.

phil-fradkin committed:
Adding updated MARA data

phil-fradkin committed:
Added calculating Mara score of a chosen cell line

phil-fradkin committed:
changed directory

sonarestis committed:
WEAVE 0.4

STRGRAPH now only contains high confidence interactions (combined score > 700)

Commit:
Visualize target and background cells

improve the definition of target/background with an interactive
component

gosuzombie committed:
refactored some code, added more comments to GSA, example output given

biodim committed:
Re-Added Deleted Files

SKamdar1 committed:
Readme

Current for visualization and PubmedMiner submodules.

SKamdar1 committed:
Readme

For Normalize_TF.

SKamdar1 committed:
Readme

For Normalize_TF.

anamqu committed:
Added files via upload

anamqu committed:
Added files via upload

anamqu committed:
Added files via upload

anamqu committed:
Added files via upload

anamqu committed:
Update Visualization.R

anamqu committed:
Update README.md

anamqu committed:
Update README.md

anamqu committed:
Added files via upload

anamqu committed:
Update README.md

anamqu committed:
Update Gene_cov.R

anamqu committed:
Update README.md

anamqu committed:
Update README.md

thejmazz committed:
comments; docs; add createCOdat.R

thejmazz committed:
Update README.md

thejmazz committed:
add design to phylify readme

SKamdar1 committed:
Vignette Rough Draft

Template of the vignette rough draft!

SKamdar1 committed:
Vignette - SK

Intro, Normalize, and Fantom vignette sections now complete!

burtonlm committed:
WEAVE module vignette

burtonlm committed:
Added brief paragraph in Section 2

anamqu committed:
Added files via upload

anamqu committed:
Added files via upload

burtonlm committed:
RANK vignette

burtonlm committed:
VALIDATE vignette

burtonlm committed:
Added paragraph on Page 2

burtonlm committed:
Added missing reference Webber et al
